.github/scripts/compare_iai.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# This script should be run on the main branch, after running the iai benchmarks on the target branch.
```

Comment:
Is it not supposed to be the other way around to see the impact it made compared to master? Otherwise I would imagine it is flipped: perf gains show up as slowdowns

Reply:
the comparison was flipped as well... but yeah not so intuitive - changed it